circle.yml

```diff
@@ -0,0 +1,18 @@
+machine:
+  timezone:
+    Asia/Tokyo
+  pre:
+    - sudo groupadd nobody
+
+dependencies:
+  override:
+    - python setup.py develop
+    - pip install coverage
+
+test:
+  override:
+    - pep8 blackbird/*.py
+    - pep8 blackbird/**/*.py
```

Comment:
`blackbird/**/*.py` means `blackbird/*/*.py`, `blackbird/*/*/*.py` ... so this is not including `blackbird/*.py`.
But I think that `pep8 blackbird/*.py blackbird/**/*.py` or `pep8 **/*.py` is more pretty.
which style do you like?